site/en/blog/new-in-chrome-94/index.md

```diff
@@ -0,0 +1,183 @@
+layout: 'layouts/blog-post.njk'
+date: 2021-09-21
+authors:
+  - petelepage
+hero: 'image/0g2WvpbGRGdVs0aAPc6ObG7gkud2/bFEcnAWdzNkXaSSp0hX4.png'
+alt: >
+  New in Chrome hero logo
+tags:
+  - new-in-chrome
+  - chrome-94
+---
+
+{% YouTube id='c5XIDt90VgY' %}
+
+Here's what you need to know:
+
+* The [default color space](#canvas-colorspace) for `<canvas>` elements is
+  now formally defined  in the spec as *sRBG*, and you can change it to
+  *Display P3*.
+* [WebCodecs](#webcodecs) is a new, low level way to access built in audio and
+  video codecs, important for streaming games, video editors, and such.
+* [WebGPU](#webgpu) starts its origin trial.
+* The [PWA Summit](#pwa-summit) is coming up October 6-7.
+* And there's plenty [more](#more).
+
+I'm [Pete LePage](https://petelepage.com), working, and shooting
+from home, let's dive in and see what's new for developers in Chrome 94.
+
+## Default color space for `canvas` elements {: #canvas-colorspace }
+
+How color is rendered on screen is critical to some users. For photographers,
+print illustrators, and many others, the colors on screen need to match what's
+printed. Starting in [Chrome 94][cs-canvas], `<canvas>` elements are fully
+color managed using *sRGB*. Previously, *sRGB* was convention, but not
+explicitly defined in the spec.
+
+```js
+opts = {colorSpace:'display-p3'};
+const ctx = canvas.getContext('2d', opts);
+```
+
+More importantly - you can now specify which color space to use when creating
+a `<canvas>` rendering context 2d object, or an `ImageData` object, including
+the P3 color space.
+
+## WebCodecs {: #webcodecs }
+
+Putting video on a page is easy enough. But, if you need to do something a
+little more complex, and interact with the components of a video stream,
+it's hard! And, typically requires you to use Web Assembly to ship
+[your own codecs][wasm-av1]!
+
+But shipping your own codec, means writing code the browser already has, and
+it can't take advantage of hardware acceleration! The Web Codecs API makes it
+possible to use the media components and codecs that are already in the browser.
+
+Personally, I've always struggled to remember the correct command line switches
+to encode a video for the web, or to convert a GIF to a video file. Using
+the APIs available via WebCodecs, I could quickly build a web app that
+reads a file and exports the correct files needed for the web.
+
+<figure class="w-figure">
+  {% Img src="image/sQ51XsLqKMgSQMCZjIN0B7hlBO02/lEovMYp8oh1JSClCLCiD.png", alt="The path from a Canvas or an ImageBitmap to the network or to storage", width="800", height="393" %}
+  <figcaption class="w-figcaption">
+    The path from a <code>Canvas</code> or an <code>ImageBitmap</code> to the
+    network or to storage
+  </figcaption>
+</figure>
+
+Web apps that require full control over the way media content is processed,
+like video editors, video conferencing, streaming apps, and so on, access to
+the browsers built in media controls, this is huge.
```

Comment:
Is this what you actually said in the video? It looks like it should be this:

*For* web apps that require full control over the way media content is processed, like video editors, video conferencing, streaming apps, and so on, access to the browsers built in media controls~~, this~~ is huge.

(Double tildes are supposed to be strike through, but GitHub doesn't support them.)